unime/src/routes/credentials/[id]/OpenBadgeRenderer.svelte

```diff
@@ -24,14 +24,25 @@
 
   {#if credential.data.credentialSubject?.achievement?.criteria?.narrative}
     <div class="prose prose-sm rounded-xl bg-background p-4 dark:prose-invert">
-      <h2>{$LL.CREDENTIAL.DETAILS.CONTENTS()}</h2>
+      <h2>{$LL.CREDENTIAL.DETAILS.CRITERIA()}</h2>
       <!-- TODO Review marked vs. markdown-it and security risks. -->
       <!-- eslint-disable-next-line svelte/no-at-html-tags -->
       {@html md.render(credential.data.credentialSubject.achievement.criteria.narrative)}
     </div>
   {/if}
 
-  <TextFieldRenderer key={'Achievement name'} value={credential.data.credentialSubject?.achievement?.name} />
+  {#if credential.data.credentialSubject?.achievement?.alignment?.length > 0}
+    <!-- Loop over the alignment array and render each item with a subtle dividing line -->
+    {#each credential.data.credentialSubject.achievement.alignment as alignItem}
+      <div class="prose prose-sm rounded-xl bg-background p-4 dark:prose-invert">
+        <!-- Render targetName in bold -->
+        <strong>{@html md.render(alignItem.targetName)}</strong>
+        {@html md.render(alignItem.targetDescription)}
+      </div>
+    {/each}
+  {/if}
+
+  <TextFieldRenderer key={'Achievement type'} value={credential.data.credentialSubject?.achievement?.achievementType} />
```

Comment:
`achievementType` is not mandatory. Please render conditionally (only if not undefined).

Reply:
fixed